src/components/Layout/Header/NavBar/FiatRamps.tsx

```diff
@@ -18,12 +20,30 @@ export const FiatRamps = (props: FiatRampsProps) => {
   const handleWalletModalOpen = () =>
     dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
 
+  const bg = useColorModeValue(
+    'linear-gradient(126deg, var(--chakra-colors-gray-100) 0%, var(--chakra-colors-gray-100) 40%, var(--chakra-colors-white) 50%, var(--chakra-colors-gray-100) 60%)',
+    'linear-gradient(126deg, var(--chakra-colors-gray-700) 0%, var(--chakra-colors-gray-700) 40%, var(--chakra-colors-gray-500) 50%, var(--chakra-colors-gray-700) 60%)',
+  )
+
+  const rainbow = keyframes`
+    0% {
+      background-position: 10% 0%;
+    }
+
+    100% {
+      background-position: 91% 100%;
+    }
+  `
+
   return (
     <Box {...props}>
       <Tooltip label={translate('fiatRamps.headerLabel')}>
         <IconButton
           icon={<FaCreditCard />}
           data-test='fiat-ramps-button'
+          bg={bg}
+          backgroundSize='300% 300%'
+          animation={`${rainbow} 3s ease infinite`}
```

Comment:
I don't mind it animating once, but it's a little annoying/distracting having it loop constantly.

Reply:
Ideally it would repeat with some sort of delay. But you can't really do that easily with CSS animations. So yah possibly route change or something could do it.